src/controllers/UsersController.php

```diff
@@ -386,6 +386,42 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request
+     * @param $db
+     *
+     * @return null
+     *
+     * @throws Exception
+     *
+     */
+    public function setTrusted($request, $db)
+    {
+        if (false == ($request->getUserId())) {
+            throw new Exception("You must be logged in to change a user account", 401);
+        }
+
+        $user_mapper = $this->getUserMapper($db, $request);
+        if (!$user_mapper->isSiteAdmin($request->getUserId())) {
+            throw new Exception("You must be an admin to change a user's trusted state", 401);
+        }
+
+        $userId = $this->getItemId($request);
+        if (!is_bool($trustedStatus = $request->getParameter("trusted", null))) {
```

Comment:
What are you checking here? ```is_bool``` checks whether the parameter is either ```true``` or ```false```. But ```$request->getPArameter()``` always returns a string… So ```is_bool``` should always fail here… Or am I missing something? 

I'd add a check whether 'trusted' is either "true" or "false" (or "yes" or "no")